### PR TITLE
code clean - qaic software kit gh action + resnet50 gh action update

### PR DESCRIPTION
**Pull request by anandhu-eng**

### CODE REVIEW

File: .github/workflows/test-mlperf-inference-mlcommons-cpp-resnet50.yml [hunk 1]
```diff
@@ -36,6 +36,26 @@ jobs:
         cm pull repo --url=${{ github.event.pull_request.head.repo.html_url }} --checkout=${{ github.event.pull_request.head.ref }}
         cm run script --quiet --tags=get,sys-utils-cm
         cm run script --quiet --tags=install,prebuilt,llvm --version=${{ matrix.llvm-version }}
-    - name: Test MLPerf Inference MLCommons C++ ResNet50
+    - name: Test MLPerf Inference MLCommons C++ ResNet50 on ${{ matrix.os }}
+      if: matrix.os == 'windows-latest'
+      run: |
+        cmr "app mlperf inference mlcommons cpp" --submitter="MLCommons" --hw_name=gh_${{ matrix.os }} --adr.loadgen.tags=_from-pip --pip_loadgen=yes -v --quiet
```
Comment (arjunsuresh) [inline .github/workflows/test-mlperf-inference-mlcommons-cpp-resnet50.yml:42]: Is pip install of loadgen enough to use loadgen in C++? Also, we don't have an automatic way to install C++ compiler on windows right?

Comment (arjunsuresh) [reply .github/workflows/test-mlperf-inference-mlcommons-cpp-resnet50.yml:43]: I think we can install llvm on Linux using
choco install llvm

Comment (anandhu-eng) [reply .github/workflows/test-mlperf-inference-mlcommons-cpp-resnet50.yml:43]: > I think we can install llvm on Linux using choco install llvm

Hi @arjunsuresh , did you mean on windows? Also, when using Chocolatey on Windows, do users need to run the command in PowerShell or Command Prompt opened with `administrative` privileges?

Comment (arjunsuresh) [reply .github/workflows/test-mlperf-inference-mlcommons-cpp-resnet50.yml:43]: @anandhu-eng I'm not sure. But no need to worry about it now - we don't need to support windows.

**Pull request merged**